src/dataflow/src/sink/kafka.rs

```diff
@@ -254,7 +348,8 @@ where
                             }
                         };
 
-                        let buf = encoder.encode_unchecked(connector.schema_id, diff_pair);
+                        let buf =
+                            encoder.encode_unchecked(connector.schema_id, diff_pair, Some(time));
```

Comment:
lets consider not emitting the timestamp if we don't want consistency info (we may need that for unit tests)